drivers/ethernet/eth_enc28j60.c

```diff
@@ -520,19 +520,28 @@ static int eth_enc28j60_rx(struct device *dev)
 		u8_t info[RSV_SIZE];
```

Comment:
Please rework the commit subject and body. No need to have "..." there, just state what is being fixed.

The commit subject could be something like `drivers: eth: enc28j60: Fix read error when ERDPT < ERXRDPT`
And the body could say something like this
```
After enc28j60 has read first packet and if ERDPT is smaller
than ERXRDPT, then this will cause read error.
The fix is to set ERDPT properly before reading next packet.

Fixes #9537

Signed-off-by: lgl88911 <lgl88911@163.com>
```

I am not sure about the commit body, as I am not familiar about what ERDPT and ERXRDPT do, so please fix if my interpretation is not correct.

Reply:
@jukkar: Your revised subject is the same as the original meaning. I'll use the subject, thanks

Reply:
You need to fix the commits in your own tree, squash them together and then force push your branch back to github. Just editing the text in github does nothing.

Reply:
I'm not familiar with git, I push the branch again, Could you help see if this is the right commit?Thanks!